src/server/protocol.ts

```diff
@@ -625,6 +626,28 @@ export interface GetMoveToRefactoringFileSuggestions extends Response {
     };
 }
 
+/**
+ * Request refactorings at a given position post pasting text from some other location.
+ */
+
+export interface GetPasteEditsRequest extends Request {
+    command: CommandTypes.GetPasteEdits;
+    arguments: GetPasteEditsRequestArgs;
+}
+
+export type GetPasteEditsRequestArgs = FileRequestArgs & {
+    pastedText: string[];
+    pasteLocations: TextSpan[];
+    copiedFrom?: { file: string; range: TextSpan[]; };

```

Comment:
Can there be multiple possible files that are being copied from? (niche, I know)

Reply:
No—I guess you were out when we discussed this, but we redesigned the protocol again specifically to prevent the API from being able to represent that.